docs/v0.6/en/reference/sql/copy.md

```diff
@@ -23,6 +23,7 @@ data (`/xxx/xxx/output.parquet` in this case).
 | Option  | Description  | Required |
 |---|---|---|
 | `FORMAT` | Target file(s) format, e.g., JSON, CSV, Parquet  | **Required** |
+| `START_TIME`/`END_TIME`| The time range within which data should be exported. `START_TIME` is inclusive and `END_TIME` is exclusive. | Optional |
```

Comment:
What happen if START_TIME <= END_TIME?

I suppose it would be an empty selection, but will it generate a warning?

Reply:
> What happen if START_TIME <= END_TIME?
> 
> I suppose it would be an empty selection, but will it generate a warning?

Currently it will create an invalid timestamp range, thus it will not come into effect. But I think maybe we should reject this statement instead?

Reply:
Silently invalid value is error-prone.

At least a warning and clearly fallback to empty range (empty to none). But better to be an error.

Not a blocker for merging this patch. Let's track an issue at the DB repo.

Reply:
> Silently invalid value is error-prone.
> 
> At least a warning and clearly fallback to empty range (empty to none). But better to be an error.
> 
> Not a blocker for merging this patch. Let's track an issue at the DB repo.

I will draft a pr to address this issue.